themes/dark/assets/js/app.js.map

```diff
@@ -1 +0,0 @@
AR,eAAAC,OAA6BH,EAAUI,OAE3C1B,EAAA,QAAIsB,UAAUA,EAAUI,KAAMJ,MAK9BT,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,EACAA,EAAA,GAGaQ,QAAQ,SAAAC,GACrBtB,EAAA,QAAIsB,UAAUA,EAAUI,KAAMJ,KAGlC,IAAMK,EAAM,IAAI3B,EAAA,SACZ0B,KAAM,MACNE,MAAAlB,EAAA,EACAmB,OAAAlB,EAAA,EACAS,YACIU,SAAArB,EAAA,GAEJsB,KAPgB,WAQZ,OACIC,eAAe,EACfC,eAAe,IAGvBC,SAAUC,OAAOC,OAAOD,OAAAlC,EAAA,EAAAkC,EAAU,OAAQ,eAC1CE,QAdgB,WAmBZ,GAJIzB,EAAA,GACAW,QAAQe,IAAI,iBAGXC,SAASC,SAASC,SAASC,SAAS,UAAW,KACxCC,EAAWC,KAAXD,OACRA,EAAOE,SAAS,SAAWC,SAAU3B,OAAO2B,WAC5CH,EAAOE,SAAS,aAEZjC,EAAA,GACAW,QAAQe,IAAI,mBAIzBS,OAAO,aAEKhD,EAAA","file":"js/app.js","sourcesContent":["import Vue from 'vue';\nimport Vuex, { mapState } from 'vuex';\nimport VueRouter from 'vue-router';\nimport AsyncComputed from 'vue-async-computed';\nimport ToggleButton from 'vue-js-toggle-button';\nimport Snotify from 'vue-snotify';\nimport Truncate from 'vue-truncate-collapsed';\nimport store from './store';\nimport router from './router';\nimport { isDevelopment } from './utils';\nimport {\n    AnidbReleaseGroupUi,\n    AppHeader,\n    AppLink,\n    Asset,\n    Backstretch,\n    Config,\n    DisplayShow,\n    FileBrowser,\n    LanguageSelect,\n    NamePattern,\n    PlotInfo,\n    RootDirs,\n    ScrollButtons,\n    SelectList,\n    ShowSelector\n} from './components';\n\nVue.config.devtools = true;\nVue.config.performance = true;\n\nVue.use(Vuex);\nVue.use(VueRouter);\nVue.use(AsyncComputed);\nVue.use(ToggleButton);\nVue.use(Snotify);\n\n// Load x-template components\nwindow.components.forEach(component => {\n    if (isDevelopment) {\n        console.debug(`Registering ${component.name}`);\n    }\n    Vue.component(component.name, component);\n});\n\n// Global components\nconst globalComponents = [\n    AnidbReleaseGroupUi,\n    AppHeader,\n    AppLink,\n    Asset,\n    Backstretch,\n    Config,\n    DisplayShow,\n    FileBrowser,\n    LanguageSelect,\n    NamePattern,\n    PlotInfo,\n    RootDirs,\n    ScrollButtons,\n    SelectList,\n    ShowSelector\n];\n\nglobalComponents.forEach(component => {\n    Vue.component(component.name, component);\n});\n\nconst app = new Vue({\n    name: 'App',\n    store,\n    router,\n    components: {\n        Truncate\n    },\n    data() {\n        return {\n            globalLoading: false,\n            pageComponent: false\n        };\n    },\n    computed: Object.assign(mapState(['auth', 'config']), {}),\n    mounted() {\n        if (isDevelopment) {\n            console.log('App Mounted!');\n        }\n\n        if (!document.location.pathname.includes('/login')) {\n            const { $store } = this;\n            $store.dispatch('login', { username: window.username });\n            $store.dispatch('getConfig');\n\n            if (isDevelopment) {\n                console.log('App Loaded!');\n            }\n        }\n    }\n}).$mount('#vue-wrap');\n\nexport default app;\n"],"sourceRoot":""}
```

Comment:
You still removed the map files

try:
```
git checkout develop --themes/**/*.map
```
and commit them :)

Reply:
Since the files updated how are the maps going to work?